### ISSUE TITLE
Update WebRTC stack to the latest stable version and Support AV1 

### ISSUE THREAD

**Issue by mekya** [ant-media/Ant-Media-Server#3076]

**Comment by dmriding** [comment]
Any update on stereo audio support?

**Comment by mekya** [comment]
Hi @dmriding ,

We'll revisit stereo audio support in this issue. It's likely that we can start this issue one month later. 

Sorry for that.

**Comment by zaleschoenborn** [comment]
any updates on stereo support?

**Comment by slodhi786** [comment]
haven't seen any update yet. is there a fix?

**Comment by mekya** [comment]
Thank you follow up guys. 

Unfortunately, for the overload of the work and unexpected case, we could not start this one. We plan to start working on this at the end of this month. We expect it to be available for beta testing late Feb.  It will be available for production in March.

**Comment by juggernautdev** [comment]
I am interested in this also

**Comment by evgenirusev** [comment]
A fix for this would help us out a lot.

**Comment by mekya** [comment]
Hi Guys,
Thank you @juggernautdev @evgenirusev  for follow up

It's one of the first items in our backlog for now. We're going to start this implementation in a couple of weeks.  

Regards,
A. Oguz

**Comment by mekya** [comment]
Current status: There is a working webrtc stack in development environment. 

Next task is patch the optimization to the webrtc stack

**Comment by mekya** [comment]
Next task is give a working solution that provides stereo support

**Comment by fedezubo** [comment]
hello, any updates on this?

**Comment by mekya** [comment]
Hi Guys,
We have a snapshot release that supports stereo publishing and a high sample rate. We've delivered it to some users and they've confirmed that it's working. 

If you're interested, I'm happy to make a demonstration and share the snapshot. 
You can book a meeting from here https://calendly.com/mekya/ or timing does not work for you, send an email to contact@antmedia.io to schedule a meeting that fits you.

**Comment by mekya** [comment]
Definitely, I would love to make the demonstration to you @juggernautdev .

You're the best ;)
